packages/store/src/PackedCounter.sol

```diff
@@ -1,127 +1,139 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.0;
 
-import { Bytes } from "./Bytes.sol";
-
 // - 7 bytes accumulated counter
 // - 5 bytes length per counter
```

Comment:
should we add an expressive comment like in `hexToPackedCounter` at the top of this file too?

Reply:
added